app/views/administrate/application/index.html.erb

```diff
@@ -55,6 +55,15 @@ It renders the `_table` partial to display details about the resources.
   </div>
 </header>
 
+<% page.scope_groups.each do |group| %>
+<div class="scope-buttons">
```

Comment:
Let’s de-couple these styles specifically from the _scope_ buttons and instead create a more generic, reusable component. Maybe something like `button-group`…? Then we can use the same styles elsewhere, when we need a grouping of buttons, again.